ams/CleanupCommand.cs

```diff
@@ -0,0 +1,233 @@
+﻿using AMSMigrate.Ams;
+using AMSMigrate.Contracts;
+using AMSMigrate.Transform;
+using Azure;
+using Azure.Core;
+using Azure.ResourceManager.Media;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Logging;
+using Microsoft.Identity.Client.Extensions.Msal;
+using Spectre.Console;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
All these system namespaces are imported automatically in .net, there is no need to add them here explicitly.